### PR TITLE
Tox runs tests, Travis runs Tox

### PR DESCRIPTION
**Pull request by hobbestigrou**
As a developer, in order to make it easier to maintain the project, I'd like to use Tox. As an example, it would make testing several Python versions easier.

### CODE REVIEW

**Comment by benoitbryon** [comment]
:+1: thanks @hobbestigrou

File: demo/setup.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def read_relative_file(filename):
 README = read_relative_file('README')
 VERSION = '0.1'
 PACKAGES = ['demoproject']
-REQUIRES = ['django-generic-filters', 'mock',
+REQUIRES = ['django-generic-filters', 'mock', 'Django<1.7'
```
Comment (Natim) [inline demo/setup.py]: Should we just put django there and test different versions of Django in the tox matrix?

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27, flake8
```
Comment (Natim) [inline tox.ini]: Do we want to start supporting Py34 and Django>1.7

Comment (benoitbryon) [inline tox.ini]: why not, good idea, but let's do it later in another PR ;)

**Pull request merged**